docker-compose.yml

```diff
@@ -21,15 +22,28 @@ services:
       - build/docker.env
     environment:
       - JOBS_DB_HOST=db
+      - JOBS_DB_USER=rudder
+      - JOBS_DB_PORT=5432
+      - JOBS_DB_DB_NAME=jobsdb
+      - JOBS_DB_PASSWORD=password
+      - DEST_TRANSFORM_URL=http://d-transformer:9090
+      - CONFIG_BACKEND_URL=https://api.rudderlabs.com
+      - WORKSPACE_TOKEN=<your_workspace_token>
+      - STATSD_SERVER_URL=metrics-exporter:9125
```

Comment:
Why do we need those? Since they are specified here: `build/docker.env`

Reply:
I think we should do not maintain both rudder-docker.yaml and docker-compose.yaml. This is only to trigger that discussion. I think we should keep these two files aligned and eventually to get rid of rudder-docker.yaml in favour of docker-compose.yaml. Plus for local development we should introduce a docker-compose.dev.yaml according to the best practices and what we do wit the value files in helm charts.

Reply:
to ease the process I reverted these changes since they may be considered out of the scope of the specific PR